src/router/Router.jsx

```diff
@@ -8,17 +8,19 @@ import FAQ from "../pages/FAQ";
 import PrivateRoute from "./PrivateRoute";
 import Login from "../pages/Login";
 import { AuthContextProvider } from "../context/AuthContext";
+import PageNotFound from "../pages/PageNotFound/PageNotFound";
```

Comment:
:wrench: change to `import PageNotFound from "../pages/PageNotFound";`

Reply:
Addressed: d612efc22a699d66c1ab524fa76d86e4b6c65096